src/Calendar.js

```diff
@@ -23,7 +24,16 @@ export default class Calendar extends Component {
 
   static defaultProps = {
     weekOffset: 0,
-    renderDay: day => day.format('YYYY-MM-D')
+    renderDay: day => day.format('YYYY-MM-D'),
+    renderHeader: (date, handlePrevMonth, handleNextMonth) => (
```

Comment:
I suggest changing this to `renderHeader: ({ date, handlePrevMonth, handleNextMonth })`.

Reply:
Fixed